cli/src/local/index.js

```diff
@@ -202,10 +202,8 @@ class Local implements Side {
       next => {
         if (doc.md5sum != null) {
           // TODO: Share checksumer instead of chaining properties
-          this.watcher.checksumer.push(tmpFile, function (err, md5sum) {
-            if (err) {
-              next(err)
-            } else if (md5sum === doc.md5sum) {
+          this.watcher.checksumer.push(tmpFile).catch(next).then(md5sum => {
```

Comment:
:warning: next may be called twice with this code : the catch returns a promise which is always resolved.

The correct way to transform a promise into a callback is to use Promise#asCallback

Reply:
My bad, I messed up trying to use `fromCallback` instead of `asCallback` and gave up since I didn't understand why it was not working.

- [ ] Use `Promise#asCallback()`